### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Srisuma13**
Removing unnecessary colons and making the text look uniform under the features section.

### CODE REVIEW

**Review by withshubh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -43,19 +43,19 @@ ToolJet is an **open-source low-code framework** to build and deploy internal to
 
 
 
-## All features
+## List of features available:
```
Comment (withshubh) [inline README.md:46]: Title change is not required. Please revert.

File: README.md [hunk 2]
```diff
@@ -43,19 +43,19 @@ ToolJet is an **open-source low-code framework** to build and deploy internal to
...
 
 - *Visual app builder:* 35+ built-in responsive widgets such as Tables, Charts, Lists, Forms, Progressbars, and more.
-- *20+ data sources:* connect to databases, cloud storages and APIs.
-- *Desktop & mobile*: ;layout widths can be customised to support different screens.
+- *20+ Data sources:* connect to databases, cloud storages and APIs.
```
Comment (withshubh) [inline README.md:49]:
```suggestion
- *20+ data sources:* connect to databases, cloud storages and APIs.
```

Comment (withshubh) [inline README.md:49]:
```suggestion
- *20+ data sources:* connect to databases, cloud storages and APIs.
```

File: README.md (hunk 1 above)
Comment (withshubh) [inline README.md:46]: Title change is not required. Please revert.

File: README.md (hunk 2 above)
Comment (withshubh) [inline README.md:49]: Please keep data lower case. Please revert this.

**Review by withshubh — CHANGES_REQUESTED** [review]
Please revert the changes that I mentioned.

File: README.md (hunk 1 above)
Comment (Srisuma13) [reply README.md]: done!

**Comment by withshubh** [comment]
Thanks! LGTM 🚀

**Review by withshubh — APPROVED** [review]

**Pull request merged**